core-projects/_index.md

```diff
@@ -23,7 +23,7 @@ Core Projects.
 Core Projects endorse SPEC documents.
 During the endorsement stage of the SPEC process, Core Project contributors
 propose, discuss, and review SPEC documents with the goal of developing
-an implementation plan that all the Core Projects will adopt and endorse.
+a coherent implementation plan suitable for all the Core Projects.
```

Comment:
I thought it felt a little circular to refer to endorsement in the endorsement process. Since *adopt* and *endorse* have specific meaning in the context of the SPEC process, I thought less-specific wording was a bit more flexible, but YMMV.